.python-version

```diff
@@ -0,0 +1 @@
+3.9.10
```

Comment:
You should really add some words in the commit message why this is done. I've no clue how this the problem works around.

Reply:
Good point 🤔  this is just a first workaround for those using pyenv on macos (should be fixed for all envirnments in #1019)